encoding/legacy-mb-schinese/gbk/gbk-decode-chinese.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+<head>
+<meta charset="utf-8"/>
+<title>chinese decoding</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<link rel="author" title="Richard Ishida" href="mailto:ishida@w3.org">
+<link rel="help" href="https://encoding.spec.whatwg.org/#names-and-labels">
+<meta name="assert" content="The browser produces the same decoding behavior for a document labeled 'chinese' as for a document labeled 'gbk'.">
+<style>
+ iframe { display:none }
+</style>
+<script src="gbk_index.js"></script>
+<script src="gbk-decoder.js"></script>
+</head>
+
+<body onload="showNodes();">
+
+<iframe src="gbk_chars-chinese.html" name="scriptWindow" id="scrwin"></iframe>
+
+<p class="instructions">Test passes if the number of failures is the same as for test gbk-decode.html</p>
```

Comment:
This makes these manual tests. I don't think that's a good idea. Why would there be failures to begin with?